src/org/openlcb/OpenLcb.java

```diff
@@ -12,9 +12,11 @@ public interface OpenLcb {
      */
      
     static final int MTI_INITIALIZATION_COMPLETE     = (int)MessageTypeIdentifier.InitializationComplete.mti();
+    static final int MTI_INITIALIZATION_COMPLETE_SIMPLE = (int)MessageTypeIdentifier.InitializationCompleteSimple.mti();
     
     static final int MTI_VERIFY_NID                  = 0x10A4;
     static final int MTI_VERIFIED_NID                = 0x10B0;
+    static final int MTI_VERIFIED_NID_SIMPLE         = 0x10B1;
```

Comment:
these values are wrong. I believe they are from pre-2012 when the MTI numbers were different.
I hope they are not used in the code...

Reply:
As near as I can tell these values and the associated `getMTI()` method aren't used anywhere.  I'll remove them at some point.